lib/matplotlib/ticker.py

```diff
@@ -1214,12 +1206,40 @@ class EngFormatter(Formatter):
          24: "Y"
     }
 
-    def __init__(self, unit="", places=None):
+    def __init__(self, unit="", places=None, sep=" "):
+        """
+        Parameters
+        ----------
+        unit : str (default: "")
+            Unit symbol to use, suitable for use with single-letter
+            representations of powers of 1000. For example, 'Hz' or 'm'.
+
+        places : int (default: None)
+            Precision with which to display the number, specified in
+            digits after the decimal point (there will be between one
+            and three digits before the decimal point). If it is None,
+            falls back to the floating point format '%g'.
```

Comment:
Actually, if I am correct, "%g" results in up to 6 *significant* digits. This means that with it, the equivalent value for ``places`` will be between 0 and 5. I'll try to make the docstring clearer.